src/GovUk.Education.ExploreEducationStatistics.Admin/Controllers/Api/UpdateReleaseRequest.cs

```diff
@@ -5,14 +5,22 @@
 using GovUk.Education.ExploreEducationStatistics.Common.Converters;
 using GovUk.Education.ExploreEducationStatistics.Common.Extensions;
 using GovUk.Education.ExploreEducationStatistics.Common.Model;
+using GovUk.Education.ExploreEducationStatistics.Content.Model;
 using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
 using static System.Globalization.CultureInfo;
 using static GovUk.Education.ExploreEducationStatistics.Content.Model.NamingUtils;
 using static GovUk.Education.ExploreEducationStatistics.Common.Services.TimePeriodLabelFormatter;
 
 namespace GovUk.Education.ExploreEducationStatistics.Admin.Controllers.Api
 {
-    public class UpdateReleaseSummaryRequest
+    public enum PublishMethod
+    {
+        Scheduled,
+        Immediate
```

Comment:
I can think of this immediate option potentially causing some problem if they can also move status back from Approved (EES-957) in future while EES-994 is hanging around, depending on how it works with amendments.

I think we will need to make sure that nobody can unapprove a release that's already published and force them to create a new amendment, otherwise reapproving with Immediate will lead to the scenario mentioned in EES-994 where data is unavailable.

I've added a comment to EES-957.

Reply:
Yep valid point. Thanks!